### ISSUE TITLE
CommandBar accessibility: No "menuitem" elements will violate rule aria-required-children for "menubar"

### ISSUE THREAD

**Issue by Leiquan-Pan** [microsoft/fluentui#12461]
### Environment Information

- **Package version(s)**: >=7.24.0 <8.0.0
- **Browser and OS versions**: Chrome 80 on Windows 10

### Describe the issue:

Hi, I'm from Microsoft Power Automate team, we are facing an accessibility issue towards CommandBar. In one of our page, we have a commandBar without any menu item only search box. It violates one accessibility rule: **aria-required-children: Ensures elements with an ARIA role that require child roles contain them**. 

It indicates that the "menubar" element requires children elements like menuitem, menuitemradio, menuitemcheckbox. I have seen that the "menubar" role property is hard-coded. I suppose it's a good idea to only have "menubar" role while there are menu items.  

#### Actual behavior:

CommandBar with no menu items failed the accessibility test, due to it violates rule: aria-required-children.

#### Expected behavior:

CommandBar should be OK with no menu items.

**Comment by aneeshack4** [comment]
@Leiquan-Pan I believe the appropriate guidance for CommandBar is to only use components with role "menuitem" set on them. I've spoken with the component owner about this before and he said the role of the CommandBar has been explicitly set as menubar as the role and it's intentional to not let uses override it. So it's up to the user to make sure that you set the role "menuitem" on whatever components you use (and choose only those that can have this role) inside the CommandBar - as shown in the [examples page](https://developer.microsoft.com/en-us/fluentui#/controls/web/commandbar).
